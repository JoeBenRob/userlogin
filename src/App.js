import React from 'react';
import './App.css';
import Nav from './Components/Nav/Nav.js';

function App() {
  return (
    <div className="App">
      <header className="App-header">User Register</header>
      <div>
        <Nav />
      </div>
    </div>
  );
}

export default App;
