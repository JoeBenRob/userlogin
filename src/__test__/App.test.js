import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';

import App from '../App';

Enzyme.configure({ adapter: new Adapter() })

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});


it('snapshot App', () => {
  // const wrapper = shallow(<App />);
  const tree = renderer.create(<App />).toJSON()
  expect(tree).toMatchSnapshot();
})