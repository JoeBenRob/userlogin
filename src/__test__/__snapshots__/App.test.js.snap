// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot App 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    User Register
  </header>
  <div>
    <div>
      <div>
        <div>
          <div>
            <div
              className="CreateForm"
            >
              <form
                onSubmit={[Function]}
              >
                <label>
                  Username: 
                  <input
                    name="name"
                    type="text"
                  />
                </label>
                <br />
                <label>
                  Password: 
                  <input
                    name="password"
                    type="password"
                  />
                </label>
                <br />
                <button
                  type="submit"
                >
                  Login
                </button>
                <p
                  style={
                    Object {
                      "color": "red",
                    }
                  }
                >
                  
                </p>
                <p>
                  
                </p>
              </form>
            </div>
          </div>
          <ul>
            <button>
              <a
                href="/UserForm"
                onClick={[Function]}
              >
                User Creation
              </a>
            </button>
            <button>
              <a
                href="/UserList"
                onClick={[Function]}
              >
                User List
              </a>
            </button>
          </ul>
        </div>
        <div />
      </div>
    </div>
  </div>
</div>
`;
